scripts/monitoring/cron-send-docker-metrics.py

```diff
@@ -22,9 +23,11 @@
     zs = ZaggSender()
     try:
         cli = AutoVersionClient(base_url='unix://var/run/docker.sock')
+        cli_storage = Client(base_url='unix://var/run/docker.sock')
         du = DockerUtil(cli)
+        Deadcontainers = cli_storage.containers(quiet=True,filters={'status':'exited'}) #get all the exited container
```

Comment:
let's change "Deadcontainers" to "dead_containers"  We don't typically use camelCase for variables names.